tutorials/Tutorial11_Pipelines.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>"Directed Acyclic Graph (DAG)" is more common, I believe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>